### DISCUSSION TITLE
[Feature request] Delay notifications

### DISCUSSION THREAD

**Discussion by haeiven** [grafana/grafana#63939] [Alerting]
Hello,

Actually, in your notification policies you have an option "Repeat interval" to prevent being spammed every minute for an alert.

Something cool we could have, is an auto delay for this. after waiting for the Repeat interval once, you can add +X minutes delay to your notification, reducing the spam :)

Exemple Scenario:
00:00 - Alert sent
00:30 - Alert sent
01:10 - Alert sent (+10m delay)
01:50 - Alert sent (+10m delay)
etc...

Best regards,

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82806. Feel free to continue the discussion around this there. Thank you!
